main.go

```diff
@@ -1,9 +1,42 @@
-package api
+package main
 
 import (
 	"fmt"
 )
 
+// ElapsedDays prints the number of days since the 1st of January of that year.
+func ElapsedDays(month, day, year int) {
+	elapsed := 0
```

Comment:
I think you can receive invalid values for the arguments of this function.

Example: `13, 32, -1`